cabinet.py

```diff
@@ -0,0 +1,260 @@
+def get_all():
+    """
+    Get all the items from the vault without the 'content' (i.e: only name and
+    tags).
+
+    :returns: The list of items
+    :ptype: List of Dictionaries.
+    """
+    global vault_config
+    cab = vault_config.get('cabinet')
+    return cab.get_all()
+
+
+def get_item(name):
+    """
+    Get an item from the vault.
+
+    :param name: The name of the item to recover.
+    :ptype: String
+
+    :returns: The item with the specified name.
+    :ptype: Dictionary
+    """
+    global vault_config
+    cab = vault_config.get('cabinet')
+    return cab.get(name)
+
+
+def add_item(item):
+    """
+    Add an item to the vault.
+
+    :param item: The item to be added.
+    :ptype: Dictionary
+    """
+    global vault_config
+    cab = vault_config.get('cabinet')
+    cab.add(item)
+
+
+def tags(value):
+    """
+    Parse the value to recover a list of tags.
+
+    :param value: The value to parse
+    :ptype: String
+
+    :returns: A list of strings composed by spliting the value by ','.
+    :rtype: List
+    """
+    try:
+        return value.split(',')
+    except:
+        raise argparse.argumenttypeerror("Multiple tags should be separated by\
+                                          comma")
+
+
+def tuple(value):
+    """
+    Parse the value to recover a tuple
+
+    :param value: The value to parse
+    :ptype: String
+
+    :returns: A tuple of two values composed by spliting the value by ','.
+    :rtype: Tuple
+    """
+    try:
+        key, value = value.split(',')
+        return key, value
+    except:
+        raise argparse.argumenttypeerror("coordinates must be x,y,z")
+
+
+class CabinetController(CementBaseController):
+    class Meta:
+        label = 'base'
+        description = "Cabinet's cli client for managing valuts."
+        arguments = [
+            (['-v', '--version'], dict(action='version', version=BANNER))
+        ]
+
+    @expose(hide=True)
+    def default(self):
+        """Without any parameter, the command will print the help"""
+        app.args.print_help()
+
+
+class ItemController(CementBaseController):
+    class Meta:
+        label = 'item'
+        stacked_on = 'base'
+        stacked_type = 'nested'
+        arguments = [
+            (['name'],
+             dict(help='The item name.', action='store')),
+            (['-t', '--tag'],
+             dict(help='Add a tag to the item', action='append')),
+            (['--tags'],
+             dict(help='Add multiple separated comma tags',
+                  type=tags, action='store')),
+            (['--content'],
+             dict(help='Add content to the item',
+                  type=tuple, action='append'))
+        ]
+
+    @expose(help='Get an item from the vault.')
+    def get(self):
+        """Get an item from the vault"""
+        name = self.app.pargs.name
+        if name:
+            self.app.log.debug('Looking for item with name "{0}"'.format(name))
+            account_id = 'my-name@my-company.com'
```

Comment:
either now that we are hardcoding credentials, or later that we may read some information from a config file and/or asking the user for it... we shouldn't repeat this all over the code.
I'd factor out all the credential stuff... maybe move it along `open_vault`?